### PR TITLE
feat: dashboard v0.2

### PR DESCRIPTION
**Pull request by rabbyOnTheMars**

### CODE REVIEW

File: src/constant/index.ts [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
+import IconCeloChainLogo from 'ui/assets/dashboard/selectChain/celo.svg';
 import IconMoonriverLogo from 'ui/assets/chain-logos/movr.svg';
 import IconMoonriverWhiteLogo from 'ui/assets/chain-logos/movr-white.svg';
+import IconMoonriverChainLogo from 'ui/assets/dashboard/selectChain/moonriver.svg';
 import IconCronosLogo from 'ui/assets/chain-logos/cronos.svg';
 import IconCronosWhiteLogo from 'ui/assets/chain-logos/cronos-white.svg';
+import IconCronosChainLogo from 'ui/assets/dashboard/selectChain/cronos.svg';
 import IconBobaLogo from 'ui/assets/chain-logos/boba.png';
 import IconBobaWhiteLogo from 'ui/assets/chain-logos/boba-white.svg';
+import IconBobaChainLogo from 'ui/assets/dashboard/selectChain/bola.svg';
```
Comment (vvvvvv1vvvvvv) [inline src/constant/index.ts:46]: can you rename `bola.svg` to `boba.svg`?

File: src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        setTimeout(() => {
+          setDrawerAnimation('fadeOutDrawer');
+        }, 100);
+      }
+    }
+  }, [showDrawer]);
+  const directionPanelData = [
+    {
+      icon: IconSendToken,
+      content: 'send',
```
Comment (vvvvvv1vvvvvv) [inline src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx:251]: Send

File: src/constant/index.ts (hunk 1 above)
Comment (rabbyOnTheMars) [reply src/constant/index.ts]: done

File: src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx (hunk 2 above)
Comment (rabbyOnTheMars) [reply src/ui/views/Dashboard/components/ChainAndSiteSelector/index.tsx]: done

**Pull request merged**